src/pkg/leaderElection.go

```diff
@@ -43,8 +44,26 @@ func RunLeaderElection(client *clientset.Clientset, lockName, lockIdentity, lock
 				log.Info().Msgf("Perform Migration")
```

Comment:
This log line needs more description - we'll probably have lost mental context when reading this in the logs.

Reply:
That's a good point, I've already lost mental context on what this line is saying. lol

Reply:
Maybe something like 
```
log.Info().Msgf("Started Leading ...")
```

Reply:
Yeah, that's already on line 46 which is why I was wondering why this was here.  I'll go ahead and remove it for now.